cdb2api/cdb2api.c

```diff
@@ -4746,30 +4746,26 @@ static int cdb2_run_statement_typed_int(cdb2_hndl_tp *hndl, const char *sql,
     PRINT_AND_RETURN(-1);
 }
 
-static char *cdb2_type_str(int type)
-{
-    switch (type) {
-    case CDB2_INTEGER:
-        return "CDB2_INTEGER";
-    case CDB2_REAL:
-        return "CDB2_REAL";
-    case CDB2_CSTRING:
-        return "CDB2_CSTRING";
-    case CDB2_BLOB:
-        return "CDB2_BLOB";
-    case CDB2_DATETIME:
-        return "CDB2_DATETIME";
-    case CDB2_INTERVALYM:
-        return "CDB2_INTERVALYM";
-    case CDB2_INTERVALDS:
-        return "CDB2_INTERVALDS";
-    case CDB2_DATETIMEUS:
-        return "CDB2_DATETIMEUS";
-    case CDB2_INTERVALDSUS:
-        return "CDB2_INTERVALDSUS";
-    default:
+const char *cdb2_tp_str[] = {
```

Comment:
I'd really rather not expose more cdb2api internals if it can be helped.  The types are exposed in cdb2api.h, maybe have a similar function/array there?

Reply:
Not sure why i can't see Nirbhay's suggestion anymore--to make the array static:
static const char *cdb2_tp_str[] = { ... }
would that be ok with you? 
I'm not sure what moving this function or the array to the .h accomplishes.

Reply:
Sure.  I didn't want to expose more cdb2api innards - that was my only concern.

Reply:
K, i understand we don't want to add random functions to the api. This can be implemented separately by client program. Thanks.